docs/bylaws.md

```diff
@@ -0,0 +1,397 @@
+# Article I: Name and Office
```

Comment:
For some reason this page isn't generating the secondary table of contents (which should appear in the upper right hand portion of the page) that lists the sections & subsections of just this page. Instead all of the many sections continue to show up in the lefthand sidebar.